src/store/actions/index.js

```diff
@@ -35,37 +40,29 @@ export const fetchUser = () => (dispatch) => {
 
 // Fetches all users
 export const fetchUsers = () => (dispatch) => {
-  axios
+  axiosWithAuth()
     .get(`${BACKEND_URL}/api/admin/users/`)
     .then((response) => dispatch({ type: 'SET_USERS', payload: response.data }))
     .catch(() => toast.error('There was a problem fetching users.'));
 };
 
 // Logs out user
 export const logOut = (history) => (dispatch) => {
-  axios
-    .get(`${BACKEND_URL}/api/auth/logout`)
-    .then((response) => {
-      localStorage.removeItem('id');
-      history.push('/welcome');
-      toast.success('You have been successfully logged out. See ya later!');
-    })
-    .catch(() =>
-      toast.error('Uh oh. You have not been successfully logged out.')
-    );
+  localStorage.removeItem('token');
+  history.push('/');
```

Comment:
This does successfully logout a user, by removing the token. But we should also invalidate the token to fully ensure a complete logout. 

Also should this push to `./welcome` ?